CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 * Bumps the minimum version of Node to `v10`
 * Removes build targets of `0.10`, `6`, and `8`, the library is now only published under a single bundled package `easypost`
+* Removes `@easypost/build` and instead uses Webpack standalone to build the project
+    * The project is now only built for `node` and may not provide browser compatible code. If you require browser compatible Javascript, we suggest you build from source
```

Comment:
```suggestion
* The project is now only built for `node` and may not provide browser compatible code. If you require browser compatible Javascript, we suggest you build from source
```
remove extra spaces.

Reply:
This is meant to be like this - it's a sub-bullet of the above item.